handlers/_swagger.js

```diff
@@ -252,12 +253,15 @@ function registerRoute(app, auth, additionalMiddleware, method, path, data, allo
             if (responseModelValidationLevel) {
                 var responseSender = res.send;
                 res.send = function (body) {
+                    var isJson = typeof body === 'string'; //body can come in as JSON or object, we want object
+                    body = isJson ? JSON.parse(body) : body;
                     var validationErrors, invalidBody;
+                    var validateResponse = validateResponseModels.bind({},res, body, data, logger, swaggerDoc);
```

Comment:
this logic will be greatly simplified in https://github.com/ryshep111/blueoak-server/tree/response-validation-level-fail